host_vars/fffw-lebenshilfe-core/base.yml

```diff
@@ -0,0 +1,8 @@
+---
+location: fffw-lebenshilfe
+role: corerouter
+target: "ipq40xx/generic"
```

Comment:
its not necessary anymore to set the target per host. Nowadays this mapping went to model group_vars. e.g. https://github.com/freifunk-berlin/bbb-configs/blob/master/group_vars/model_avm_fritzbox_4040.yml#L5

.. It works by using 2 inventory stages. First one creating the group assignments based on host_vars, and a second one which can create group assignments of group_vars which were referenced in the first run.
https://github.com/freifunk-berlin/bbb-configs/tree/master/inventory

Reply:
Good to know. Hopefully I will get some time and adjust the examples in the documentation accordingly.

Reply:
target in host-vars removed.